packages/react/src/components/OverflowSet/OverflowButton.tsx

```diff
@@ -77,7 +102,7 @@ export const OverflowButton = (props: IOverflowSetProps) => {
 
           // Add this persisted keytip to our internal list, use a temporary uniqueID (its content)
           // uniqueID will get updated on register
-          persistedKeytips[persistedKeytip.content] = persistedKeytip;
+          persistedKeytips[unregisteredKeytipID + persistedKeytip.content] = persistedKeytip;
```

Comment:
Seems a little odd that we have to do this. Is there another way we can detect if a keytip has been registered?

Reply:
I could have had a separate data structure that holds a queue of unregistered keytips, and then that queue is flushed in the "effect" function. two ways to skin a cat imo

Reply:
this just feel a little more fragile/hacky compred to the explicit queue approach

Reply:
added a queue instead